app/src/features/submissions/components/SubmissionDataGrid/SubmissionDataGrid.tsx

```diff
@@ -0,0 +1,78 @@
+import { Divider, Paper, Toolbar } from '@mui/material';
+import Typography from '@mui/material/Typography';
+import { Box } from '@mui/system';
+import { DataGrid, GridRowSelectionModel } from '@mui/x-data-grid';
+import { SubmissionFeatureRecordWithTypeAndSecurity } from 'interfaces/useSubmissionsApi.interface';
+import React, { useState } from 'react';
+import useColumns from './useSubmissionDataGridColumns';
```

Comment:
nit and possibly subjective: Can we import this as `useSubmissionDataGridColumns` just to make it easy to track what this is later on in the code?

Reply:
updated